scripts/pages/leaderboards/leaderboards.ts

```diff
@@ -25,9 +25,13 @@ class LeaderboardsHandler {
 		$.RegisterEventHandler('Leaderboards_TimesFiltered', $.GetContextPanel(), (count) =>
 			this.onTimesUpdated(count)
 		);
-		$.RegisterForUnhandledEvent('EndOfRun_Show', (reason) => this.onShowEndOfRun(reason));
-		$.RegisterForUnhandledEvent('Leaderboards_MapDataSet', (isOfficial) => this.onMapLoad(isOfficial));
-		$.RegisterForUnhandledEvent('Leaderboards_MapLeaderboardsLoaded', (map) => this.onMapLeaderboardsLoad(map));
+		$.RegisterEventHandler('EndOfRun_Show', $.GetContextPanel(), (reason) => this.onShowEndOfRun(reason));
+		$.RegisterEventHandler('Leaderboards_MapDataSet', $.GetContextPanel(), (isOfficial) =>
+			this.onMapLoad(isOfficial)
+		);
+		$.RegisterEventHandler('Leaderboards_MapLeaderboardsLoaded', $.GetContextPanel(), (map) =>
+			this.onMapLeaderboardsLoad(map)
+		);
```

Comment:
These (and above line) should use `this.panels.cp`. Not that perf is a big deal, but it's good to be consistent.